examples/client_test.cpp

```diff
@@ -1696,14 +1696,17 @@ MAGNETURL is a magnet link
 			{
 				for (announce_entry const& ae : h.trackers())
 				{
+					auto best_ae = std::min_element(ae.endpoints.begin(), ae.endpoints.end()
```

Comment:
Yeah, I must admit I wrote this just to make the problem go away for now with the least amount of effort. We can go back later and enhance the tracker display to handle multiple endpoints better.